packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -56,13 +56,59 @@ import HbarLimit from '../hbarlimiter';
 import constants from './../constants';
 import { SDKClientError } from './../errors/SDKClientError';
 import { JsonRpcError, predefined } from './../errors/JsonRpcError';
+import {RelayImpl} from "../relay";
 
 const _ = require('lodash');
 
 export class SDKClient {
     static transactionMode = 'TRANSACTION';
     static queryMode = 'QUERY';
     static recordMode = 'RECORD';
+
+    static initClient(logger: Logger, hederaNetwork: string, type: string | null = null): Client {
+        let client: Client;
+        if (hederaNetwork in RelayImpl.chainIds) {
+            client = Client.forName(hederaNetwork);
+        } else {
+            client = Client.forNetwork(JSON.parse(hederaNetwork));
+        }
+
+        if (type === 'eth_sendRawTransaction') {
+            if (
+                process.env.OPERATOR_ID_ETH_SENDRAWTRANSACTION &&
+                process.env.OPERATOR_KEY_ETH_SENDRAWTRANSACTION
+            ) {
+                client = client.setOperator(
+                    AccountId.fromString(
+                        process.env.OPERATOR_ID_ETH_SENDRAWTRANSACTION
+                    ),
+                    PrivateKey.fromString(
+                        process.env.OPERATOR_KEY_ETH_SENDRAWTRANSACTION
+                    )
+                );
+            } else {
+                logger.warn(`Invalid 'ETH_SENDRAWTRANSACTION' env variables provided`);
+            }
+        } else {
+            if (process.env.OPERATOR_ID_MAIN && process.env.OPERATOR_KEY_MAIN) {
+                client = client.setOperator(
+                    AccountId.fromString(process.env.OPERATOR_ID_MAIN.trim()),
```

Comment:
Of all these things you're pulling out of `env` why is this one the only one that needs a `trim()`?  (Just curious.)

Reply:
Good question, and the honest answer is I don't know. these code looks new, but is in fact "moved" from one class to another. so I only kept it as I found it  😅